assembly-syntax/src/ast/constants.rs

```diff
@@ -325,3 +342,27 @@ impl fmt::Display for ConstantOp {
         }
     }
 }
+
+// HASH KIND
+// ================================================================================================
+
+/// Represents the type of the final value to which some string value should be converted.
+#[derive(Debug, Copy, Clone, PartialEq, Eq, Hash)]
+pub enum HashKind {
+    /// Reduce a string to a single felt value (unused right now)
+    Felt,
+    /// Reduce a string to a word using Blake3 hash function
+    Blake3,
+    /// Reduce a string to event (unused right now)
+    Event,
```

Comment:
Let's use `Word` here instead of `Blake3` as all these variants will use BLAKE3 for hashing.

Also, it is not clear to me if we need to add the currently unused variants - especially the `Felt` variant as I'm not sure what we'll do with it.

Reply:
Make sense. I'll remove the unused variants in that case.